src/lib/hooks/swap/useSwapInfo.tsx

```diff
@@ -42,6 +44,8 @@ function useComputeSwapInfo(): SwapInfo {
     [Field.OUTPUT]: outputCurrency,
   } = useAtomValue(swapAtom)
 
+  const feeOptions = useAtomValue(feeOptionsAtom)
```

Comment:
nit: rename to `fee` instead of `feeOptions` (and `feeAtom`), and use the shorthand when you instantiate the object below.

Reply:
up for doing the shorthand, but the sdk type calls this `FeeOptions`, so I want to stick with that for the atom and type